changelog.md

```diff
@@ -95,6 +94,7 @@
 
 
 ## Standard library additions and changes
+- Deprecated std/mersenne
```

Comment:
this goes under: `## Changes affecting backward compatibility`

Reply:
Just fixed it. Sorry.